packages/dina-ui/pages/object-store/object/view.tsx

```diff
@@ -75,7 +77,7 @@ export default function MetadataViewPage() {
           {buttonBar}
           <div className="row">
             <div className="col-md-4">
```

Comment:
I would remove this `col-md-4` className. This restricts the size of the preview to be  only about a quarter of the screen. 

Before:

![image](https://user-images.githubusercontent.com/3820842/159078302-a3e6b109-5e37-4041-8b6a-22eb16118113.png)

After:

![image](https://user-images.githubusercontent.com/3820842/159078383-40ad38b0-d540-4080-8f0c-5cb4f544b4b0.png)

Then the image will always display as the true size and also in the center 👍